wbooks/src/app/screens/SearchBook/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { FlatList } from 'react-native';
+import NoneResult from '@components/NoResults';
+import RenderBookList from '@components/RendererBookList';
+import useFilterBook from '@hooks/useFilterBook';
+import keyExtractor from '@utils/keyExtractor';
+
+import styles from './styles';
+
+export default function SearchBook() {
+  const { filteredBooks } = useFilterBook();
+  return (
+    <>
+      {filteredBooks.length === 0 && <NoneResult />}
```

Comment:
i think this should work too
```suggestion
      {filteredBooks.length && <NoneResult />}
```

Reply:
not worked for my, but i replaced it by const function